farm/data_handler/processor.py

```diff
@@ -491,6 +493,32 @@ def _sample_to_features(self, sample) -> dict:
         )
         return features
 
+class TextPairClassificationProcessor(TextClassificationProcessor):
```

Comment:
Please add a short docstring

Reply:
It should be clear what input format is expected (e.g. colum names = "text"; "text_b", "label")